src/transformers/generation/logits_process.py

```diff
@@ -133,14 +133,60 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 class MinNewTokensLengthLogitsProcessor(LogitsProcessor):
     r"""
     [`LogitsProcessor`] enforcing a min-length of new tokens by setting EOS (End-Of-Sequence) token probability to 0.
+    Note that `min_length` will compute the length of `prompt + newly generated tokens` whereas `min_new_tokens` will
```

Comment:
> will compute the length of `prompt + newly generated tokens`

This is only true for decoder-only models like Llama 2. On other models, the two logit processors behave the same way. Let's clarify this in the docstring :)

Reply:
Thank you for the clarification!